server/handlers/default_test.go

```diff
@@ -172,6 +177,52 @@ func TestGetKeyHandlerCreatesOnce(t *testing.T) {
 	}
 }
 
+// Rotating a key for a non-supported role results in a 400.
+func TestRotateKeyHandlerInvalidRole(t *testing.T) {
+	state := defaultState()
+	for _, role := range []string{data.CanonicalRootRole, data.CanonicalTargetsRole, "targets/a", "invalidrole"} {
```

Comment:
These are awesome!  Non-blocking:  wondering if we could unify this with the `TestGetKeyHandlerInvalidRole` the same way you did with `TestKeyHandlersNoRoleOrRepo`?   And apologies there was no existing test for getting a key with an invalid algorithm, but was wondering if that test could also be adapted to test both handlers?